report_external_layout_simple/data/report_layout_data.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="report_layout_simple" model="report.layout">
+        <field name="name">Simple</field>
+        <field name="sequence">6</field>
+        <field name="view_id" ref="report_external_layout_simple.external_layout_simple"/>
+        <field name="image">/report_external_layout_simple/static/img/preview_simplified.png</field>
+        <field name="pdf">/report_external_layout_simple/static/pdf/preview_simplified.pdf</field>
```

Comment:
I think pdf previews are ignored. But now its somewhat dynamically generated via preview (old versions only had fixed pdf preview). 
Though old version had this data, but it was never specified to be loaded (I guess it was disabled, because not worked?).

And yes I tested, it works correctly. You can even change some colors for invoice via preview. Previews left as I see standard does same thing (but maybe that is their own leftover too).